hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/BlockManagerImpl.java

```diff
@@ -103,6 +108,15 @@ public long putBlock(Container container, BlockData data,
   public static long persistPutBlock(KeyValueContainer container,
       BlockData data, ConfigurationSource config, boolean endOfBlock)
       throws IOException {
+    boolean incrementalEnabled =
+            config.getBoolean(OZONE_CHUNK_LIST_INCREMENTAL,
```

Comment:
The indent here is 8. It's expected to be 4.  It's odd that there is no checkstyle warning.